models/JobApplicantsModel.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+const { constants } = require("../helpers/constants");
+
+const Schema = mongoose.Schema;
+const DataTypes = Schema.Types;
+
+const applicantSchema = new Schema({
+    candidate: {
+        type: DataTypes.ObjectId,
+        required: true
+    },
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 4 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
	},
```